src/software/pipeline/main_imageMatching.cpp

```diff
@@ -92,51 +92,59 @@ std::ostream& operator<<(std::ostream& os, const OrderedPairList & pl)
   return os;
 }
 
-enum class EImageMatchingMultiSfM
+enum class EImageMatchingMode
 {
   A_AB,
-  A_B
+  AAB,
+  AB,
+  A
 };
 
 
 /**
- * @brief get informations about each EImageMatchingMultiSfM
+ * @brief get informations about each EImageMatchingMode
  * @return String
  */
-std::string EImageMatchingMultiSfM_description()
+std::string EImageMatchingMode_description()
 {
-  return "The mode to combine matching between multiple SfM: \n"
-         "* a_ab : image matching for images in input SfMData A plus between A and B\n"
-         "* a_b  : image matching between input SfMData A and B\n";
+  return "The mode to combine matching: \n"
+         "* a+ab : image matching of images in SfMData A plus image matching between images of A and B\n"
+         "* aab  : image matching of images in SfMData A and between images of A and B\n"
+         "* ab   : image matching between images of SfMData A and B\n"
+         "* a    : image matching of images in SfMData A";
```

Comment:
```
return "The mode to combine image matching between the input SfMData A and B: \n"
           "* a|a+a|b : A with A + A with B\n"
           "* a|ab  : A with A and B\n"
           "* a|b   : A with B\n"
           "* a|a    : A with A";
```

Reply:
@fabiencastan I try it but, the pipe character gives a lot of trouble for the command line.

Reply:
If it's inside quotes, it should work.
But you can use `/` instead.